dataSources/srd/spells.json

```diff
@@ -1675,6 +1675,22 @@
 			"concentration": false,
 			"material": "a tiny ball of bat guano and sulfur"
 		}
+	},
+		{
+		"castingTime": "action",
+		"description": "You hurl a mote of fire at a creature or objeet within range. Make a ranged spell attack against the target. On a hit, the target takes 1dlO fire damage. A flammable object hit by this spell ignites if it isn't being worn or carried. This spell's damage increases by 1dlO when you reach",
```

Comment:
"objeet" here should be "object", and "1dlO" should be "1d10". Additionally, the description is cut off here after "This spell's damage increases by 1d10 when you reach".